services/httpd/handler.go

```diff
@@ -594,6 +595,19 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user meta.U
 
 	epoch := strings.TrimSpace(r.FormValue("epoch"))
 
+	timeFormats := []string{
+		"rfc3339",
+		"epoch",
+	}
+	// timeFormat should default to "epoch"
+	timeFormat := strings.TrimSpace(r.FormValue("time_format"))
+	if timeFormat == "" {
+		timeFormat = "epoch"
+	} else if !slices.Contains(timeFormats, timeFormat) {
+		h.httpError(rw, `time format must be one of the following: "epoch", "rfc3339"`, http.StatusBadRequest)
```

Comment:
Take advantage of the `timeFormats` that's already defined (and fix the parameter name). Example:
```go
h.httpError(rw, fmt.Sprintf("time_format must be one of the following: %s", strings.Join(timeFormats, ",")), http.StatusBadRequest)
```